src/pages/CreateProtocolPage.jsx

```diff
@@ -69,7 +69,20 @@ function CreateProtocolPage(props) {
             <NavBar />
             <div className="container-fluid d-flex flex-column flex-grow-1 font-barlow p-4 p-lg-5">
                 <div className="row m-0">
-                    <h1 className="font-century-gothic color-grey fs-3 fw-bold p-0 pb-4 pb-lg-5 m-0">Gerador de formulários</h1>
+                    <div className="col-4">
+                        <h1 className="font-century-gothic color-grey fs-3 fw-bold p-0 pb-4 pb-lg-5 m-0">Gerador de formulários</h1>
+                    </div>
+                    <div className="col-4">
+                    </div>
+                    <div className="col-2 ">
+                        <TextButton type="submit" hsl={[6, 84, 75]} text="Estatísticas" />
+                    </div>
+                    <div className="col-2">
+                        <TextButton type="submit" hsl={[37, 98, 76]} text="Respostas" />
+                    </div>
```

Comment:
```suggestion
                    <div className="col-2 ">
                        <TextButton role="link" hsl={[6, 84, 75]} text="Estatísticas" />
                    </div>
                    <div className="col-2">
                        <TextButton role="link" hsl={[37, 98, 76]} text="Respostas" />
                    </div>
```

- Como são links que não submetem formulário, não cabe type="submit". Já role="link" teoricamente deve ajudar a indexação e entendimento da página pelos navegadores.